hack/tool/tackle

```diff
@@ -74,17 +74,16 @@ def getKeycloakToken(host, username, password, client_id='tackle-ui', realm='tac
 
 def apiJSON(url, token, data=None, method='GET', ignoreErrors=False):
     debugPrint("Querying: %s" % url)
-    match method:
```

Comment:
The match-case works in Python >=3.10, rewritten to if-elif-else to ensure compatibility with older Python versions.